proposals/tf_utf8_identifiers/README.md

```diff
@@ -0,0 +1,435 @@
+
+Finally, other languages do not have standard Romanizations, and in general a Romanized character does not have a 1:1 correspondence with its source character.  For example, the Jyutping Romanization "sam1" maps to several unrelated Cantonese characters such as 心, 森, and 深.  This further fences off content creators the world over from leveraging and improving upon USD as the canonical mechanism for scene interchange across an ever-expanding ecosystem of tooling and pipelines in the virtual space.
+
+# An Overview of Identifiers in USD
+
+Identifier syntax is currently enforced in USD by several different mechanisms.  The primary mechanism is the inlined function `TfIsValidIdentifier`, which states that "an identifier is valid if it follows the C/Python identifier convention; that is, it must be at least one charaacter long, must start with a letter or underscore, and must contain only letters, underscores, and numerals".  However, there are several other code paths that validate different flavors of identifiers, including namespaced and variant identifiers, that use similar logic to achieve validation.  In particular, this means that any change to the logic governing core identifier validation in `TfIsValidIdentifier` necessitates change in several different similar validation functions across the `Sdf` module (for example, `SdfPath::IsValidNamespacedIdentifier`).
+
+True identifier validation in USD must at a minimum support the following general use cases:
+
+- Validation of identifiers (and dependent constructs, such as paths) from both text and binary usd files
+- Validation of identifiers via direct ask in code (e.g. `SdfPath::IsValidIdentifier`)
+- Usage of (and indirectly validation of) identifiers in scene navigation such as `GetPrimAtPath`
+- Generation of appropriate identifiers for schema extensions that can be appropriately compiled in code-based schemas
+
+Furthermore, these use cases must be supported at both the C++ and Python layers.
+
+It is also necessary to define what we mean by *identifier*.  The current implementation mixes two syntactically similar but semantically different concepts:
+
+- **Prim names:** The name of a USD prim, differentiating one sibling prim from another and forming the hierarchical path from which a prim can be referenced
```

Comment:
I think this is a good point and worth capturing, but for focus and given that the proposal isn't as this time recommending that prim names extend beyond identifiers, it might be worth condensing or moving to a sibling document.

Reply:
Captured in the comment below.